src/uct/ib/base/ib_device.c

```diff
@@ -420,13 +420,11 @@ static int uct_ib_device_cleanup_proc(void* arg)
 {
     uct_ib_device_nb_close_ctx *ctx = arg;
     static const char *process_name = "ucx_cleanup";
-    void *dfl_action = NULL;
+    void *dfl                       = SIG_DFL;
```

Comment:
shouldn't we pass struct sigaction?

Reply:
Yep, https://elixir.bootlin.com/linux/v4.4/source/arch/x86/include/uapi/asm/signal.h#L102